ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,8 @@
 ## Instruction on how to do issues like your boy darkosto
  
  ### Issue / bug
- - Explain the issue / bug as easily understandably as possible.
+ - Explain the issue / bug as understandably as possible.
+ - If possible add minecraft, forge, java and modpack version numbers for debugging (mod authors might need these).
  - If the issue/bug is recurring explain how it can be recreated (helps darkosto!).
- - Add a error/crash log to [Pastebin](https://pastebin.com/), and link it here, if possible(if you disslike pastebin any other similiar site is fine too)
- - Add a video/gif of the issue/bug if possible (makes it easier for darkosto to understand you).
\ No newline at end of file
+ - If possible add error/crash log to [Pastebin](https://pastebin.com/) and link it here (if you disslike pastebin any other similiar site is fine too).
```

Comment:
_Sorry for this second review request but GitHub sometimes **dislikes** what I do._
`disslike` should be `dislike`

Reply:
Oh, i swear i fixed that